Woof/Woof/Model/Walking/WalkingStatus.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+/**
+ The possible walking status values for a `Walking`.
```

Comment:
nit:
```suggestion
The current walking status.
```

Reply:
I guess that enum describes all possible variants, not a current value of walking status.